docs/source/config_file.rst

```diff
@@ -254,8 +254,8 @@ For more information, see the :ref:`None and optional handling <none-and-optiona
 section of the command line docs.
 
 ``no_implicit_optional`` (bool, default False)
-    Changes the treatment of arguments with a default value of None by not implicitly
-    making their type Optional.
+    Changes the treatment of arguments with a default value of ``None`` by not implicitly
```

Comment:
I suppose the verbatim formatting was forgotten here, so added the `Optional` ref as well.

Reply:
+1